Jenkinsfile.cd

```diff
@@ -383,6 +384,48 @@ def ubuntuTesting() {
     }
 }
 
+def androidTesting() {
+    def libindyVersion
+
+    node('ubuntu'){
+        stage('Android test'){
+            try{
+                echo 'Building android binaries. This will act as a minimal test for android'
+                checkout scm
+
+                libindyVersion = getSrcVersion("libindy")
+                def libnullpayVersion = getSrcVersion("libnullpay")
+                echo 'Building android binaries: Building docker image'
+                buildEnv = dockerHelpers.build('indy-sdk', 'libindy/ci/ubuntu.dockerfile libindy/ci')
+                sh 'chmod -R 777 libindy/'
+                sh 'chmod -R 777 libindy/ci/'
+
+                //TODO: Enable emulator test when the unit test start passing on android device
```

Comment:
What status of this TODO? Do we plan to merge it?

Reply:
Several unit tests of libindy are failing on Android. I suspect these are failing mostly because of the unavailability of the internet permissions to binary. My effort was mainly focused on executing the tests on Android regardless of the outcome of the tests.
Because of the failing unit tests, I think these tests should not run on CD pipeline without fixing them. 
Also, I noticed it is taking more than an hour to execute tests on the emulator. I have tried with increased Ram on the emulator but it does not have much effect on the time to execute tests.

Reply:
We see significant degradation on iOS emulator with a large amount of logging. And there is a trace level in current android config. May be it's a similar issue?

Reply:
Can we run some set of test (only `--lib` for example?) to be sure about this part of CI is working well

Reply:
BTW if you suspect, that the reason of failed tests is invalid permission in CI environment, the fix of this is must have part of creating the CI pipeline

Reply:
If the reason is something else than it's critical blocker for publishing any artifacts for android.

Reply:
@vimmerru FYI

Reply:
@faisal00813 We need to understand why tests fail. Do you have logs?

Reply:
After additional discussion I can suggest the next steps:
1) specify appropriate ticket in this comment, not just `//TODO ...` but `//TODO IS-XXX: ...`
2) consider this PR as minimal solution for Android CD - add appropriate comment in Jira (with the link to additional ticket)
3) Mark status of android builds as experimental until we have clear vision about the root of the problem